Packs/Whois/ReleaseNotes/1_2_2.md

```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Whois
+- Fixed a wrong related indicator type.
```

Comment:
```suggestion
- Fixed an issue where the *IP* indicator type was associated instead of the *CIDR* indicator type in the *FeedRelatedIndicators* field.
```